From 07266723119b04869956db6edcd5c986052bec9c Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Wed, 27 Aug 2014 15:37:49 +1000
Subject: [PATCH] Revert "radeon/compute: Report a value for
 PIPE_SHADER_CAP_MAX_CONST_BUFFER_SIZE"

This reverts commit f40a0750a4fcd2c7f233b962a5e74166e12456ad.
---
 src/gallium/drivers/r600/r600_pipe.c   | 11 -----------
 src/gallium/drivers/radeonsi/si_pipe.c |  7 -------
 2 files changed, 18 deletions(-)

diff --git a/src/gallium/drivers/r600/r600_pipe.c b/src/gallium/drivers/r600/r600_pipe.c
index 6ee4484..0668b27 100644
--- a/src/gallium/drivers/r600/r600_pipe.c
+++ b/src/gallium/drivers/r600/r600_pipe.c
@@ -416,17 +416,6 @@ static int r600_get_shader_param(struct pipe_screen* pscreen, unsigned shader, e
 		return 1; /* Max native address registers */
 	case PIPE_SHADER_CAP_MAX_CONSTS:
 		return R600_MAX_CONST_BUFFER_SIZE;
-	case PIPE_SHADER_CAP_MAX_CONST_BUFFER_SIZE:
-		if (shader == PIPE_SHADER_COMPUTE) {
-			uint64_t max_const_buffer_size;
-			pscreen->get_compute_param(pscreen,
-				PIPE_COMPUTE_CAP_MAX_MEM_ALLOC_SIZE,
-				&max_const_buffer_size);
-			return max_const_buffer_size;
-
-		} else {
-			return R600_MAX_CONST_BUFFER_SIZE;
-		}
 	case PIPE_SHADER_CAP_MAX_CONST_BUFFERS:
 		return R600_MAX_USER_CONST_BUFFERS;
 	case PIPE_SHADER_CAP_MAX_PREDS:
diff --git a/src/gallium/drivers/radeonsi/si_pipe.c b/src/gallium/drivers/radeonsi/si_pipe.c
index afb9bcb..ca8cea3 100644
--- a/src/gallium/drivers/radeonsi/si_pipe.c
+++ b/src/gallium/drivers/radeonsi/si_pipe.c
@@ -317,13 +317,6 @@ static int si_get_shader_param(struct pipe_screen* pscreen, unsigned shader, enu
 		switch (param) {
 		case PIPE_SHADER_CAP_PREFERRED_IR:
 			return PIPE_SHADER_IR_LLVM;
-		case PIPE_SHADER_CAP_MAX_CONST_BUFFER_SIZE: {
-			uint64_t max_const_buffer_size;
-			pscreen->get_compute_param(pscreen,
-				PIPE_COMPUTE_CAP_MAX_MEM_ALLOC_SIZE,
-				&max_const_buffer_size);
-			return max_const_buffer_size;
-		}
 		default:
 			return 0;
 		}
-- 
1.9.3

